charts/zalenium/templates/_pod-template.yaml

```diff
@@ -114,8 +114,10 @@ spec:
           value: {{ .Values.hub.keepOnlyFailedTests | quote }}
         - name: RETENTION_PERIOD
           value: {{ .Values.hub.retentionPeriod | quote }}
+        {{- if (.Values.ingress.path) and (not (eq .Values.ingress.path "/")) }}
```

Comment:
Same comment + 
Is this condition really necessary, `CONTEXT_PATH` could be equals to `/`

Reply:
Agree, I'll remove `and (not (eq .Values.ingress.path "/")`